packages/azure/data_stream/firewall_logs/elasticsearch/ingest_pipeline/azure-shared-pipeline.yml

```
@@ -0,0 +1,95 @@
+---
+description: Pipeline for parsing Azure shared log fields.
+processors:
+  - set:
+      field: cloud.provider
+      value: azure
+  - grok:
+      field: azure.resource_id
+      patterns:
+      - /SUBSCRIPTIONS/%{SUBID:azure.subscription_id}/RESOURCEGROUPS/%{GROUPID:azure.resource.group}/PROVIDERS/%{PROVIDERNAME:azure.resource.provider}/NAMESPACES/%{NAMESPACE:azure.resource.namespace}/AUTHORIZATIONRULES/%{RULE:azure.resource.authorization_rule}
+      - /subscriptions/%{SUBID:azure.subscription_id}/resourceGroups/%{GROUPID:azure.resource.group}/providers/%{PROVIDERNAME:azure.resource.provider}/namespaces/%{NAMESPACE:azure.resource.namespace}/authorizationRules/%{RULE:azure.resource.authorization_rule}
```

```suggestion
      - (?i)/subscriptions/%{SUBID:azure.subscription_id}/resourceGroups/%{GROUPID:azure.resource.group}/providers/%{PROVIDERNAME:azure.resource.provider}/namespaces/%{NAMESPACE:azure.resource.namespace}/authorizationRules/%{RULE:azure.resource.authorization_rule}
```

Could simplify the char classes below similarly. Also in the grok processors below.

---

Good suggestion - simplified in https://github.com/elastic/integrations/pull/3578/commits/8e3aecf79ddacd354a8af66aca85274512d8e9dd